src/Microsoft.CSharp/src/MultilingualResources/Microsoft.CSharp.de.xlf

```diff
@@ -0,0 +1,520 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@naamunds there was some talk about potentially having these in another parallel repo, something to consider. Not even sure if MAT actually supports that scenario. Beyond the actual size of the content do we have any ideas on the impact of the build times that this adds?